app/webpack/css/main.scss

```diff
@@ -158,4 +158,10 @@ li.list-inline-item {
       }
     }
   }
+}
+
+figure.attachment {
+  img {
+    max-width: 100%;
+  }
 }
```

Comment:
@cdoyle-temple Would you clean up the missing end of file?

Reply:
done